usbtmc/usbtmc.py

```diff
@@ -696,51 +718,77 @@ def clear(self):
             raise UsbtmcException("Clear failed", 'clear')
 
     def remote(self):
-        "Send remote command"
+        """
+        Send the remote command to the instrument.
+        :return:
+        """
         raise NotImplementedError()
 
     def local(self):
-        "Send local command"
+        """
+        Send the local command to the instrument.
+        :return:
+        """
         raise NotImplementedError()
 
     def lock(self):
-        "Send lock command"
-
+        """
+        Advantest/ADCMT ONLY: Send the lock command to the instrument.
```

Comment:
They're really more like "features" than features.  It may be best to segment that part out as _advantest_lock and _advantest_unlock for internal use only.